idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1468,6 +1468,28 @@ struct DescribeTaskListResponse {
   20: optional TaskListStatus taskListStatus
 }
 
+struct ListTaskListPartitionsRequest {
+  10: optional string domain
+  20: optional TaskList taskList
+}
+
+struct ListTaskListPartitionsResponse {
+  10: optional Partitions activityTaskPartitions
+  20: optional Partitions decisionTaskPartitions
+}
+
+struct Partitions {
```

Comment:
This struct seems a bit weird to me from composability standpoint. Here is one proposal to simplify things for clarity, let me know what you think:

```
struct TaskListPartitionMetadata {
    string name or string key
    string ownerHostName or string owner
}

struct ListTaskListPartitionsResponse {
    list<TaskListPartitionMetadata> activityTaskListPartitions
    list<TaskListPartitionMetadata> decisionTaskListPartitions
}
```

```

Reply:
Yeah, that seems more clear and possible good structure for my API information.